claim-check/code-samples/sample-2/client-consumer/EventHubsConsumer.cs

```diff
@@ -76,48 +35,78 @@ public void Configure()
 
             string storageConnectionString = ConfigurationManager.AppSettings["StorageConnectionString"];
             string eventhubConnectionString = ConfigurationManager.AppSettings["EventHubConnectionString"];
-
             downloadDestination = ConfigurationManager.AppSettings["DownloadDestination"];
-
+            string blobContainerName = ConfigurationManager.AppSettings["BlobContainerName"];
             Console.WriteLine("Connecting to Storage account...");
-
-            storageAccount = CloudStorageAccount.Parse(storageConnectionString);
-
+            BlobContainerClient blobContainerClient = new BlobContainerClient(storageConnectionString, blobContainerName);
             Console.WriteLine("Connecting to EventHub...");
-            
-            client = EventHubClient.CreateFromConnectionString(eventhubConnectionString);
+            processor = new EventProcessorClient(blobContainerClient, EventHubConsumerClient.DefaultConsumerGroupName, eventhubConnectionString);
         }
 
         public async Task ProcessMessages(CancellationToken cancellationToken)
         {
-            Console.WriteLine("The application will now start to listen for incoming message.");
+            Console.WriteLine("The application will now start to listen for incoming messages.");
+            int eventIndex = 0;
```

Comment:
What is the point of `eventIndex`?  I see how we increment it each time we get a message but I don't understand why.  It does not appear to be used anywhere?